src/ui/public/visualize/visualize.less

```diff
@@ -16,7 +16,7 @@ visualize {
     display: flex;
     flex-direction: row;
 
-    flex: 1 0;
+    flex: 1 1 0;
```

Comment:
@cjcenizal IE11 behaves exactly the other way around here. 

- `flex: 1 0;`  works on IE11
- `flex: 1 1 0;` does not work on IE11